src/platforms/web/runtime/modules/attrs.js

```diff
@@ -109,7 +109,14 @@ function baseSetAttr (el, key, value) {
       // $flow-disable-line
       el.__ieph = true /* IE placeholder patched */
     }
-    el.setAttribute(key, value)
+    // When changing the attribute 'name' of an iframe element, the iframes window must be updated aswell. Otherwise
+    // anchor tags with a 'target' attribute won't hit the right iframe.
+    if (el.tagName === 'IFRAME' && key === 'name') {
+      setTimeout((el: HTMLIFrameElement) => {
```

Comment:
Is there any reason we need a parameter instead of directly using `el` inside of the function?